content/03-reference/01-tools-and-interfaces/02-prisma-client/05-filtering.mdx

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Filtering'
+metaTitle: 'Filtering (Reference)'
+metaDescription: 'Advanced filtering use cases.'
```

Comment:
I've tried to optimize the meta descriptions also for SEO, which according to [this source](https://moz.com/learn/seo/meta-description) would be a length between 50 and 160 characters.

Also, since it's shown in Google results, I've tried to package it in a sentence that also gives some sort of TLDR of the content, e.g. [this](https://www.google.com/search?ei=FoWpXoifAoK2kwXsnJnABw&q=prisma+docs+relation+queries+reference&oq=prisma+docs+relation+queries+reference):

![image](https://user-images.githubusercontent.com/4058327/80603410-acc6e280-8a30-11ea-9876-640ae8c6e713.png)

Reply:
Done! ✅